packages/client/src/components/MovieListPage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import CategoryButton from './button/CategoryButton';
+import { apiURL } from '../apiURL';
+
+const MovieListPage = () => {
+  const [categoriesList, setCategoriesList] = useState([]);
+  const [catId, setCatId] = useState(null);
+
+  useEffect(() => {
+    const fetchCategories = async () => {
+      try {
+        const response = await fetch(`${apiURL()}/categories`);
+        const data = await response.json();
+        setCategoriesList(data);
+      } catch (error) {
+        error(error);
```

Comment:
Have you tested that the `error()` call works? Where does it print the error?

Reply:
We made changes, yeah it was not working :) thanks for your feed back :)